content/draft/general/password_strength.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Password strength spec"
+description: ""
+date: 2020-10-06T08:48:57+00:00
+lastmod: 2020-10-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "general"
+weight: 100
+toc: true
+---
+
+# Password Strength spec
+
+## Motivation
+
+It's quite important the user chooses a strong password since the entire database is encrypted using that password. No matter how strong the encryption is, if the someone is able to brute force the password, they can read the database. This is why it's important to choose a strong password and to make sure it's not easy to guess. We can aid the user to choose a strong password by providing certain criteria.
+
+## Designs
+
+* [designs](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=11%3A4921)
```

Comment:
It would be nice to have different userflows put into screenshots here for clarity and then have a resource section where we link the source file instead.

Reply:
@PascalPrecht ask and you shall be given ;-)

See userflow that starts here https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=1940%3A215207

and also the userflow that starts here https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=2201%3A235245